### DISCUSSION TITLE
Solucan lastik ip ve araba sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#253] [Soru Cevap] [answered]
Elimizde 10 cm boyunda olan ve sonsuza dek gerip uzatabileceğimiz özdeş bir lastik ip var. Bu ipin A ucunu bir duvara bağlıyoruz. B ucunu da bir arabaya bağlıyoruz. A ucuna bir solucan yerleştiriyoruz. Solucan ipin B ucuna doğru gidiyor. Saniyede 1 cm yol alıyor. Solucan 1 saniye süreyle gidince duruyor. Solucan durunca araba lastik ipi çekerek B ucunu 10 cm ileriye götürüyor ve duruyor. Araba durunca solucan yine 1 saniye süreyle gidiyor ve duruyor. Solucan durunca araba lastik ipin B ucunu yine 10 cm ileriye götürüyor ve duruyor. Ve bu olay hiç durmadan böyle tekrarlanıyor. Solucan lastik ipin B ucunu yakalayabilir mi? Yakalarsa ne zaman yakalayamazsa neden yakalayamaz?

**Comment by tolga0102** [comment, marked as answer]
```python
solucan=0
araba=10
sn=1
while solucan<=araba:
    solucan+=1
    araba+=10
    solucan*=araba/(araba-10)
    sn+=1
print(sn,"saniye sonra solucan arabayı yakalar.")
```
